code/modules/cargo/bounties/security.dm

```diff
@@ -1,93 +1,227 @@
-/datum/bounty/item/security/recharger
-	name = "Weapon Recharger"
-	description = "Nanotrasen military academy is conducting marksmanship exercises. They request that a recharger be shipped."
-	reward = CARGO_CRATE_VALUE * 4
-	required_count = 1
-	wanted_types = list(/obj/machinery/recharger = TRUE)
-
-/datum/bounty/item/security/pepperspray
-	name = "Pepperspray"
-	description = "We've been having a bad run of riots on Space Station 76. We could use some new pepperspray cans."
-	reward = CARGO_CRATE_VALUE * 6
-	required_count = 4
-	wanted_types = list(/obj/item/reagent_containers/spray/pepper = TRUE)
-
-/datum/bounty/item/security/prison_clothes
-	name = "Prison Uniforms"
-	description = "TerraGov has been unable to source any new prisoner uniforms, so if you have any spares, we'll take them off your hands."
-	reward = CARGO_CRATE_VALUE * 4
-	required_count = 4
-	wanted_types = list(/obj/item/clothing/under/rank/prisoner = TRUE)
-
-/datum/bounty/item/security/plates
-	name = "License Plates"
-	description = "As a result of a bad clown car crash, we could use an advance on some of your prisoners' license plates."
-	reward = CARGO_CRATE_VALUE * 2
-	required_count = 10
-	wanted_types = list(/obj/item/stack/license_plates/filled = TRUE)
```

Comment:
Something that comes to mind as I'm not against axing these as security officer bounties (Since, ideally, it should be prisoners that should be making these in the first place, not tempting officers to do it for some reason), making a prisoner bounty category could happen in the fullness of time

Out of scope for this PR though, I'm just yapping.